proxies/live/apiproxy/resources/jsc/SetErrorVars.js

```diff
@@ -0,0 +1,210 @@
+// Regex to validate the id is a valid UUID
+const uuidRegex = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi
+const nhsNumberRegex = /^[0-9]{10}$/gi
+const dobRegex = /^[0-9]{4}-[0-9]{2}-[0-9]{2}$/gi
+
+var data = context.getVariable("request.content.data")
```

Comment:
`context.getVariable("")` and `context.setVariable("")` is the main method of changing flow variables within the apigee flow from within a JS policy.